packages/expo-cellular/README.md

```diff
@@ -0,0 +1,23 @@
+# expo-cellular
+
+Information about the user’s cellular service provider, such as its unique identifier, cellular connection type ,and whether it allows VoIP calls on its network.
+
+# API documentation
+
+- [Documentation for the master branch](https://github.com/expo/expo/blob/master/docs/pages/versions/unversioned/sdk/cellular.md)
+
+# Installation
+
+This package is pre-installed in [managed](https://docs.expo.io/versions/latest/introduction/managed-vs-bare/) Expo projects. You may skip the rest of the installation guide if this applies to you.
+
+For bare React Native projects, you must ensure that you have [installed and configured the `react-native-unimodules` package](https://github.com/unimodules/react-native-unimodules) before continuing.
+
+### Add the package to your npm dependencies
+
+```
+npm install expo-cellular
```

Comment:
```suggestion
expo install expo-cellular
```
?

Reply:
`expo install` is only available after sdk 33 and all the docs now are using `npm install`. So maybe `npm install` is better? Or shall we leave a note about the version in `README.md`?

Reply:
we should update this across all READMEs and docs probably. i'll open an issue for it but we don't need to worry about it for this particular PR i think.